lusSTR/cli/gui.py

```diff
@@ -197,14 +255,27 @@ def interactive_plots_allmarkers(sample_df, flagged_df):
     max_yvalue = (int(math.ceil(max_reads / n)) * n) + n
     increase_value = int(math.ceil((max_yvalue / 5) / n)) * n
     n = 0
-    for marker in sample_df["Locus"].unique():
+    all_loci = f_strs if st.session_state.kit == "forenseq" else p_strs
```

Comment:
p_strs = powerseq and f_strs = forenseq? I could only figure that out by finding where they're called in the code. You could consider renaming the variables to be more self explanatory.